src/v/kafka/server/server.h

```diff
@@ -167,6 +169,10 @@ class server final : public net::server {
 
     ssx::semaphore& memory_fetch_sem() noexcept { return _memory_fetch_sem; }
 
+    handler_probe& handler_probe(api_key key) {
```

Comment:
Missing public method doc. It should mention what happens on invalid keys.

Reply:
Added a method doc to the forward declaration of the method in the header.